{
  "cveTags": [],
  "descriptions": [
    {
      "lang": "en",
      "value": "Rejected reason: DO NOT USE THIS CANDIDATE NUMBER. ConsultIDs: none. Reason: The CNA or individual who requested this candidate did not associate it with any vulnerability during 2020. Notes: none"
    }
  ],
  "id": "CVE-2020-3814",
  "lastModified": "2023-11-07T03:23:05.010",
  "metrics": {},
  "published": "2021-02-15T22:15:13.227",
  "references": [],
  "sourceIdentifier": "security@debian.org",
  "vulnStatus": "Rejected"
}